src/App.tsx

```diff
@@ -2,108 +2,34 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { User } from './types/User';
+import { Comment } from './types/Comment';
+import { Post } from './types/Post';
+import { PostList } from './components/PostList';
+
+function getUser(userId: number): User | null {
```

Comment:
will be better to name this function like `findUser`, because you can't be sure that you get some object, you can return null from this function

Reply:
but this name was here by default, so it's not critical to fix it